modules/backend/classes/NavigationManager.php

```diff
@@ -24,7 +24,7 @@ class NavigationManager
     protected $callbacks = [];
 
     /**
-     * @var array List of registered items.
+     * @var MainMenuItem[] List of registered items.
```

Comment:
I've never seen this syntax before, it's still just an array, the difference is that it's an array of MainMenuItems instead of an array of arrays.

Reply:
This syntax is a way to let your editor know which type to expect from an array. It will allow autocompletion and functions as "jump to definition". In this particular case I think it's useful in future debugging or code modifications.
See https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#arrays for more information.